### PR TITLE
Feature/hacker export

### PR DESCRIPTION
**Pull request by RohitGarudadri**
### Tickets:

- HCK-232

### List of changes:

- Hacker export returns all fields of hacker now

### Type of change:

Please delete options that aren't relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [x] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots:

There's actually only 1 resume column

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Update text for travel status: bus
 
+### Fixed
```
Comment (loreina) [inline docs/CHANGELOG.md]: move this to unreleased after you pull from `develop`

File: src/features/Search/Search.tsx [hunk 2]
```diff
@@ -175,17 +175,37 @@ class SearchContainer extends React.Component<{}, ISearchState> {
...
+      headers.push('application.shortAnswer.comments');
+      headers.push('application.shortAnswer.question1');
+      headers.push('application.shortAnswer.question2');
+      headers.push('accountId.email');
+      headers.push('application.accommodation.shirtSize');
+      headers.push('application.accommodation.impairments');
+      headers.push('application.accommodation.barriers');
+      headers.push('application.accommodation.travel');
+      headers.push('application.other.ethnicity');
+      headers.push('accountId.gender');
```
Comment (loreina) [inline src/features/Search/Search.tsx]: sponsors should be able to get gender. @isabelledelmundo @MaxMB15 can you clarify?

**Comment by pierreTklein** [comment]
Consider changing the header names if possible so that it’s the leaf-most attribute.

**Comment by krubenok** [comment]
While I appreciate the gif of the CSV it contained personal information of the hackers and needed to be removed.

File: src/features/Search/Search.tsx (hunk 2 above)
Comment (krubenok) [reply src/features/Search/Search.tsx]: Ethnicity, ShirtSize, impairments, barriers, and travel also aren't relevant to sponsors.

The essay questions from the applications and the comments section are also of questionable worth to me.

Comment (loreina) [reply src/features/Search/Search.tsx]: ... the bottom list are available for staff only.

Comment (isabelledelmundo) [reply src/features/Search/Search.tsx]: sponsors shouldn't be able to export with gender but i think sponsors should be able to export with email as part of the fields

**Comment by loreina** [comment]
@isabelledelmundo can you review also?

**Pull request merged**